### ISSUE TITLE
Wave Speed/Color—slightly off

### ISSUE THREAD

**Issue by dyunus** [1kc/razer-macos#225]
Feel like the colors used for the Wave effect don't match the colors used by synapse3, specifically the light blue color seems too dark

"Normal" speed appears to be slower too compared to synapse3

**Comment by Stanman04** [comment]
true

**Comment by dhobi** [comment]
AFAIK, Synapse 3 controls all effects by continuously sending custom frames (colors for each key) to the device. This allows for more flexible effects (i.e. stacking the effects on top of each other) and can also look different than the "hardware builtin" commands which Synapse 2 (and razer-macos) uses.

**Comment by nightsky30** [comment]
> AFAIK, Synapse 3 controls all effects by continuously sending custom frames (colors for each key) to the device. This allows for more flexible effects (i.e. stacking the effects on top of each other) and can also look different than the "hardware builtin" commands which Synapse 2 (and razer-macos) uses.

Yes :) And something witnessed more recently with openrazer is the hardware based wave effect can also vary by device.

Most devices animate the hardware based effect with a pattern that does not match Synapse 3. However, when implementing support for the newer Razer Charging Pad Chroma in openrazer, it used the same hardware based effect/command as the older devices, but DOES match the wave pattern/colors Synapse 3 uses.

Aggravating because anyone using non-Synapse software with multiple devices that set wave (via hardware effect) could have devices that don't match wave patterns.
